packages/flutter/lib/src/material/button_style_button.dart

```diff
@@ -16,6 +16,21 @@ import 'material.dart';
 import 'material_state.dart';
 import 'theme_data.dart';
 
+/// {@template flutter.material.ButtonStyleButton.iconAlignment}
+/// Determines alignment of icon i.e. after/before the label in TextButton.icon,
+/// ElevatedButton.icon, OutlinedButton.icon, FilledButton.icon,
+/// FilledButton.tonalIcon.
```

Comment:
```suggestion
/// Determines alignment of icon i.e. after/before the label in TextButton.icon,
/// ElevatedButton.icon, OutlinedButton.icon, FilledButton.icon,
/// FilledButton.tonalIcon.
```
Elaborate "Determines alignment of icon i.e. after/before the label" with more details and it should probably say something similar to "start of the button" and "end of the button"

Supported widgets could be in a bullet list

Reply:
Sure

Reply:
Will do it today

Reply:
was stuck on an unexpected mistake - https://github.com/flutter/flutter/pull/137348#discussion_r1436621698

so code-comment will take it up today.

Reply:
Done